src/Umbraco.Core/Persistence/Repositories/PartialViewRepository.cs

```diff
@@ -12,7 +11,7 @@ namespace Umbraco.Core.Persistence.Repositories
     internal class PartialViewRepository : FileRepository<string, IPartialView>, IPartialViewRepository
     {
         public PartialViewRepository(IUnitOfWork work)
-			: this(work, new PhysicalFileSystem(SystemDirectories.MvcViews + "/Partials/"))
+			: this(work, FileSystemProviderManager.Current.PartialViewsFileSystem)
```

Comment:
In this case this could certainly be injectable/testable since we are 100% in control of instantiating this instance